packages/migration/runMigrations.sh

```diff
@@ -32,3 +32,6 @@ yarn --cwd $SCRIPT_PATH migrate-mongo status --file $APP_CONFIG
 yarn --cwd $SCRIPT_PATH migrate-mongo up --file $USER_MGNT_CONFIG
 yarn --cwd $SCRIPT_PATH migrate-mongo status --file $USER_MGNT_CONFIG
 
+# performance migration
+yarn migrate-mongo up --file migrate-mongo-config-performance.js
```

Comment:
Use `--cwd $SCRIPT_PATH` here as well just like above. Otherwise the cwd path will be a bit broken if this script is ran from country-config, like it is in `restore-backup.sh`